### PR TITLE
grammatical error in the facilitators section corrected

### PR DESCRIPTION
**Pull request by Esyteck**
The screenshot below shows the errors detected in red circles and line. Kindly review

Screenshot 2022-10-16 at 22 43 07

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -351,9 +351,9 @@ This is an invitation-based role. Facilitators are offered an honourarium in rec
 
 ### Our facilitators
 
-Our facilitator are essential for the program:
+Our facilitators are essential for the program:
```
Comment (NPDebs) [inline about.md:354]: Nice catch! 👍

File: about.md [hunk 2]
```diff
@@ -351,9 +351,9 @@ This is an invitation-based role. Facilitators are offered an honourarium in rec
...
 
-- [{{ ols-3-facilitators | uniq | size }} facilitators](/ols-3#facilitators) for [**OLS-3**]({% link ols-3.md %}) ({{ ols-3-schedule.weeks['01'].start }} - {{ ols-3-end }})
```
Comment (NPDebs) [inline about.md:356]: @Esyteck These lines of code dynamically generate info about the number of facilitators involved in each cohort. This should not be classified as a typo.

However, your approach seems to cover the events of 1 or more facilitators...which is great. 👍

Let's see what @bebatut thinks.

**Review by NPDebs — COMMENTED** [review]
@Esyteck Good observation! 👍

File: about.md (hunk 2 above)
Comment (bebatut) [reply about.md:356]: The suggestion look good

**Comment by NPDebs** [comment]
> Thanks @Esyteck for spotting and fixing that, and @NPDebs for the review

My pleasure. 🙌🏼🤍

**Comment by Esyteck** [comment]
You welcome ma'am.

**Pull request merged**